### ISSUE TITLE
ssh daemon not starting, problems with libuv.so.1

### ISSUE THREAD

**Issue by marhoy** [hassio-addons/addon-ssh#123]
# Problem/Motivation

I'm running the latest version of hassio and all components.
Since updating to version 7.3.0 of this addon, the ssh-daemon dies on startup and I'm seing this in the addon-log:

```
 Add-on version: 7.3.0
 You are running the latest version of this add-on.
 System: HassOS 3.12  (armv7 / raspberrypi3)
 Home Assistant Core: 0.106.5
 Home Assistant Supervisor: 209
...
[services.d] starting services
[services.d] done.
[12:23:14] INFO: Starting the ttyd daemon...
[12:23:14] INFO: Starting the Home Assistant STDIN service...
[12:23:14] INFO: Starting the SSH daemon...
Server listening on 0.0.0.0 port 22.
Server listening on :: port 22.
Error loading shared library libuv.so.1: Exec format error (needed by /usr/bin/ttyd)
Error relocating /usr/bin/ttyd: uv_signal_start: symbol not found
Error relocating /usr/bin/ttyd: uv_signal_stop: symbol not found
Error relocating /usr/bin/ttyd: uv_timer_stop: symbol not found
Error relocating /usr/bin/ttyd: uv_poll_start: symbol not found
Error relocating /usr/bin/ttyd: uv_is_closing: symbol not found
Error relocating /usr/bin/ttyd: uv_poll_init: symbol not found
...
Error relocating /usr/bin/ttyd: uv_pipe_init: symbol not found
Received signal 15; terminating.
[cont-finish.d] executing container finish scripts...
[cont-finish.d] 99-message.sh: executing... 
[12:23:14] INFO: STDIN service was stopped, restarting...
[cont-finish.d] 99-message.sh: exited 0.
[cont-finish.d] done.
[s6-finish] waiting for services.
[s6-finish] sending all processes the TERM signal.
```

Is is possible to manually downgrade to another version of this addon?
I have ssh-access to the hassio-host at port 22222.

**Comment by marhoy** [comment]
After restoring version 7.1.0 of the addon (from a snapshot), it works as expected again.

**Comment by ironslow** [comment]
Same here, unable to start the add-on at version 7.3.0

**Comment by marhoy** [comment]
This issue is solved in version 7.3.1.
Thanks! :)
